rpcq/_server.py

```diff
@@ -35,15 +35,17 @@ class Server:
     """
     Server that accepts JSON RPC calls through a socket.
     """
-    def __init__(self, rpc_spec: RPCSpec = None, announce_timing: bool = False):
+    def __init__(self, rpc_spec: RPCSpec = None, announce_timing: bool = False,
+                 catch_exceptions: bool = True):
```

Comment:
This needs a different name, something like serialize_exceptions or return_expceptions or something else that indicates the exception gets turned into an RPC message and then packaged up to the client